src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
```

Comment:
Consider moving the 'Random' instance to be a class-level constant, since it does not need to be instantiated multiple times if the 'ColorSupplier' class is used multiple times.